linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -112,13 +113,18 @@ public static void main(String[] arguments)
   static ImmutableMap<String, String> findLocalArtifacts(
       RepositorySystem repositorySystem, RepositorySystemSession session, Path projectDirectory) {
     ImmutableMap.Builder<String, String> artifactToVersion = ImmutableMap.builder();
+
+    // Relative paths to the files in the project
     Iterable<Path> paths = MoreFiles.fileTraverser().breadthFirst(projectDirectory);
 
     for (Path path : paths) {
       if (!path.getFileName().endsWith("pom.xml")) {
         continue;
       }
 
+      Verify.verify(
```

Comment:
This will throw an exception the calling code isn't prepared to handle. This method should handle both cases here.

Reply:
Updated to use `path.relativize` in case it is an absolute path.